schema-evolution-manager.gemspec

```diff
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name              = 'schema-evolution-manager'
+  s.version           = '0.9.17'
```

Comment:
Will want to update util/create-release.rb to change the version on new releases, and also to create and publish the gem file for all new releases.